ci-operator/config/open-cluster-management/insights-client/open-cluster-management-insights-client-main.yaml

```diff
@@ -21,14 +21,22 @@ images:
       - registry.ci.openshift.org/open-cluster-management/builder:go1.16-linux
   to: insights-client
 promotion:
-  name: "2.4"
+  name: "2.5"
   namespace: open-cluster-management
 resources:
   '*':
     requests:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: pr-image-mirror
```

Comment:
pr-image-mirror is already present in this file. Please remove the duplicate addition.